### ISSUE TITLE
coredump by "less --header=1,,2 file"

### ISSUE THREAD

**Issue by Jun-T** [gwsw/less#545]
Either with less-661 or git HEAD:
`less --header=1,,2 file` coredumps at linenum.c:396.
anchor is not yet initialized when parsing the command line options.
```
#0  0x000000000041512b in find_pos (linenum=2) at linenum.c:396
#1  0x00000000004177e2 in parse_header (s=<optimized out>, 
    lines=<synthetic pointer>, cols=<synthetic pointer>, 
    start_pos=<synthetic pointer>) at optfunc.c:1052
#2  opt_header (type=<optimized out>, s=<optimized out>) at optfunc.c:1071
#3  0x00000000004182aa in scan_option (s=<optimized out>) at option.c:298
#4  0x0000000000402605 in main (argc=1, argv=0x7ffc0ab4a538) at main.c:314
```

**Comment by gwsw** [comment]
Fixed in f9e5b43d8fa6f0e04d49422165b5ede4786fc292.

**Comment by Jun-T** [comment]
Thank you!
